sos/report/plugins/kubernetes.py

```diff
@@ -247,62 +264,76 @@ def postproc(self):
         # output that is not hit by the previous iteration.
         self.do_cmd_private_sub(self.kube_cmd)
 
+        pathexp_config_files = fr'^({"|".join(self.config_files)})'
+        self.do_file_private_sub(pathexp_config_files)
+
+        # clear base64 encoded PEM from kubeconfigs files
+        regexp = r'LS0tLS1CRUdJ[A-Za-z0-9+/=]+'
+        subst = '***** SCRUBBED BASE64 PEM *****'
+
+        self.do_path_regex_sub(pathexp_config_files, regexp, subst)
+
+        pathexp_files = fr'^({"|".join(self.files)})'
+        self.do_path_regex_sub(pathexp_files, regexp, subst)
+
 
 class RedHatKubernetes(Kubernetes, RedHatPlugin):
 
-    # OpenShift Container Platform uses the atomic-openshift-master package
-    # to provide kubernetes
-    packages = ('kubernetes', 'kubernetes-master', 'atomic-openshift-master')
+    packages = KUBE_PACKAGES + (
+        'kubernetes-master',
+        'atomic-openshift-master'
+    )
 
-    files = (
+    files = KUBECONFIGS + (
         '/etc/origin/master/admin.kubeconfig',
         '/etc/origin/node/pods/master-config.yaml',
     )
 
-    kube_cmd = "kubectl"
+    services = KUBE_SVCS
+
+    def check_enabled(self):
+        if self.is_installed("openshift-hyperkube"):
```

Comment:
Why this package presence shoud disable this plugin? (I guess there is a rationale that I dont see / know)

Reply:
For what it's worth the commit title is `[kubernetes] ensure we are not enabled if Openshift is`
The idea is that we match on a lot of things that might also be present on OpenShift installs, but running both plugins is wasteful
I'll add a comment

Reply:
done